src/Data.js

```diff
@@ -0,0 +1,141 @@
+import guitar from "./assets/images/guitar.jpeg";
+import harmonium from "./assets/images/harmonium.jpeg";
+import speakers from "./assets/images/speakers.jpeg";
+import mike from "./assets/images/mike.jpeg";
+import keyboardbook from "./assets/images/keyboardbook.jpeg";
+import ARbook from "./assets/images/ARbook.jpeg";
+
+// import { v4 as uuidv4 } from "uuid";
+
+export const data = [
+  {
+    id: 1,
+    name: "Silver Enterprise Amazing Guitar",
+    description: `9 Perfect 18" 4-String Acoustic Guitar Learning Kids Toy  (Brown)`,
+    ratings: 4,
+    delivery: "Fast Delivery",
+    image: guitar,
+    priceDetails: {
+      originalPrice: 999,
+      discount: "22%  ",
```

Comment:
`discount` attribute is used for computing the total price in `totalPrice()` in `utils.js`file. Consider changing this to a number. While displaying it you can use a template string to format it as `${discount} %`